TodoList/Src/TodoList.Api/App_Start/DependencyConfig.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Web.Http;
+using TodoList.Api.Services;
+using TodoList.DI.DependencyResolvers;
+using TodoList.Repository;
+
+namespace TodoList.Api
+{
+    public class DependencyConfig
+    {
+        public static void Register(HttpConfiguration config)
+        {
+            var dependencyResolverBuilder = DependencyResolverBuilder.GetInstance();
+            config.DependencyResolver = dependencyResolverBuilder
+                .Bootstrap(new ApiBootstrapper())
+                .Bootstrap(new RepositoryBootstrapper())
```

Comment:
It's better to write:
```suggestion
                .Bootstrap<RepositoryBootstrapper>()
```
to avoid writing new() every time.

Reply:
I added this modification to the code, but now I don't know how to make the test on `Bootstrap()` method.

I tried something like this, but that is nonsense and didn't find another way.
```
            var container = Substitute.For<ITodoListContainer>();
            var bootsrapper = Substitute.For<IBootstrapper>();
            
            var builder = new DependencyResolverBuilder(container);
            
            builder.Bootstrap<typeof(bootsrapper)>();

            bootsrapper.Received(1).Register(container);
```

Is there some other way to test the `Bootstrap()` method? @HanaNavratilova

Reply:
ff71167